product-web-app/views/add-new-job-role.html

```diff
@@ -25,9 +25,18 @@
                     <label for="sharePointLink">SharePoint Link</label>
                     <input type="text" name="sharePointLink" value="{{ jobRole.sharePointLink }}" class="form-control" required>
                 </div>
+
+                <div class="form-group">
+                    <label for="capabilityId">Select desired Capability</label>
+                    <select select id="/capability" name="capabilityId" class="form-select form-select-lg mb-3 mt-2" aria-label=".form-select-lg" required>
+                    {% for Capability in Capability %}
+                        <option value="{{ Capability.id }}">{{Capability.capability}}</option>
+                    {% endfor%}
```

Comment:
```suggestion
                    {% for capability in capabilities %}
                        <option value="{{ capability.id }}">{{capability.name}}</option>
                    {% endfor%}
```

Reply:
Corrected.
But now, we will see the leader's name, and I think we are supposed to see the department?